### ISSUE TITLE
(navigation-react-native) Feature Request: Ability to enable default behavior for large title in iOS 13 or above

### ISSUE THREAD

**Issue by julian-baumann** [grahammendick/navigation#470]
### Issue Description

Since iOS 13 the default behavior of the large title is to have a transparent background and no shadow color.

See [uinavigationbar changes in ios13](https://sarunw.com/posts/uinavigationbar-changes-in-ios13/)
Would be great if there was a functionality to enable this behavior.
 
This behavior has become the standard in recent iOS versions and can be found in Apps like Settings, Apple TV, Shortcuts, Mail, Messages, Health and many more.

Wanted behavior:
![wanted-behavior](https://user-images.githubusercontent.com/39265507/96162903-6c451d80-0f19-11eb-815d-b92bf3f07e42.gif)

**Comment by grahammendick** [comment]
Hey, thanks for the request and for using the Navigation router.

I am using those new iOS13 navigation bar appearances but I’m [setting all three to the same value](https://github.com/grahammendick/navigation/blob/eda7481ed5e12e764d8defe51ba0dff9e8026d58/NavigationReactNative/src/ios/NVNavigationBarView.m#L65). But you’re saying that the default iOS behaviour is for the `scrollEdgeAppearance` to have a transparent background and the `standardAppearance` a default background, right?

That sounds like a good idea. What’s the best way to go about it? 

1. We could add a >= iOS 13 check to make it match the defaults
2. Or we could add the ability to style the large title separately

I prefer the first approach because the Navigation router aims to align with the platform defaults. Also, styling the large title separately is something I’ve been avoiding for now because it’s much harder to think how to expose this to React. What about you?

**Comment by julian-baumann** [comment]
Hey, 
the first approach sounds good to me too. 
We don't even have to check for the iOS version. We could just get rid this line `self.reactViewController.navigationItem.scrollEdgeAppearance = appearance`.

Since `scrollEdgeAppearance` is a iOS 13+ feature and without assigning `appearance` to it, it just uses the default settings, which would be the desired behavior.

But maybe there are some people who like the present behavior. So I would suggest to add a prop like `useScrollEdgeAppearance` and only apply `self.reactViewController.navigationItem.scrollEdgeAppearance = appearance` if the variable is `true`.

**Comment by julian-baumann** [comment]
Also just wanted to say you build a great navigation library!
It uses real native components with modern features and it is also an absolute pleasure to program with :)

**Comment by grahammendick** [comment]
Thanks, that's kind of you!

The more I think about this, the more I think it's a bug rather than a feature request. The Navigation router aims to keep to the platform defaults and this just got missed. So I don't think we need.a prop to keep the present behaviour.

> self.reactViewController.navigationItem.scrollEdgeAppearance = appearance.

I just tried getting rid of that line and you're right, it does just work. I'm surprised because I thought it wouldn't keep the large title attributes like color and font - but it did! Do you know why? Are they taken from the standardAppearance?

**Comment by julian-baumann** [comment]
Yeah I think standardAppearance is used here. If is explained in the apple docs regarding scrollEdgeAppearance

```
If the value of this property is nil, UIKit uses a modified version of the settings found in the standardAppearance property.

```

https://developer.apple.com/documentation/uikit/uinavigationbar/3198027-scrolledgeappearance

**Comment by grahammendick** [comment]
Thanks, I'm happy with your fix then. We should delete this line too for consistency
```obj-c
self.reactViewController.navigationItem.compactAppearance = appearance;
```
You deserve the kudos for finding the fix so would you like to submit a PR with it in?

**Comment by julian-baumann** [comment]
Thanks I can do it if you want, but I am not at home at the moment. I think it would take like 10 hours till I can do it. So I wouldn't be sad or something if you want to do it immediately. :)
Since it is only a very minor change.

**Comment by grahammendick** [comment]
There's no rush. Whenever you get round to it is fine by me

**Comment by julian-baumann** [comment]
Alright, thank you. And also for your quick responses

**Comment by grahammendick** [comment]
Fixed by #471
